scripts/configs/example2.yaml

```diff
@@ -0,0 +1,36 @@
+# An example configuration file showing how to define multiple approach/env
```

Comment:
give these files better names like `example_basic` and `example_multiple_combinations`